samples/javascript_nodejs/12.nlp-with-luis/deploymentScripts/msbotClone/bot.recipe

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
recipe file has changed to include additional stuff by default.  Replace all "<%= botName %>" references in the below json with "nlp-with-luis"

{
  "version": "1.0",
  "resources": [
    {
      "type": "endpoint",
      "id": "1",
      "name": "development",
      "url": "http://localhost:3978/api/messages"
    },
    {
      "type": "luis",
      "id": "34",
      "name": "<%= botName %>-LUIS"
    },
    {
      "type": "endpoint",
      "id": "2",
      "name": "production",
      "url": "https://your-bot-url.azurewebsites.net/api/messages"
    },
    {
      "type": "abs",
      "id": "3",
      "name": "<%= botName %>-abs"
    },
    {
      "type": "appInsights",
      "id": "4",
      "name": "<%= botName %>-insights"
    },
    {
      "type": "blob",
      "id": "5",
      "name": "<%= botName %>-blob",
      "container": "botstatestore"
    }
  ]
}